bin/git-theta

```diff
@@ -95,7 +96,9 @@ def add(args):
     model_path = git_utils.get_relative_path_from_root(repo, args.file)
     theta_model_dir = git_utils.get_git_theta_model_dir(repo, model_path, create=True)
 
-    param_dict = checkpoints.PyTorchCheckpoint.from_file(args.file)
+    checkpoint_type = args.type or "pytorch"
```

Comment:
Add `# TODO: Don't default to pytorch once we have other checkpoint formats supported`

Reply:
Updated